README.md

```diff
@@ -287,10 +287,37 @@ npm run format
 
 Note that this project has configuration for [pre-commit](https://github.com/pre-commit/pre-commit). To set up locally:
 
-```bash
+````bash
 # if you don't have it yet, globally
 pip install pre-commit
 
+`pylint`, which is run as part of `pre-commit`, relies on the poetry packages all being installed. If you are running this on your local machine you need to install them if you have not done so previously. Poetry automatically creates a virtual environment when you do this, which the `pylint` command will make use of
+
+```bash
+# if you haven't run this locally previously
+poetry install
+````
+
+`pylint` also relies on the [libpq](https://www.postgresql.org/docs/9.5/libpq.html) library being installed as a global package on your local machine. The installation commands below are for a mac.
+
+```bash
+brew install libpq
+
+# update your path
```

Comment:
not everyone uses zsh. I suggest changing this to "follow the homebrew post-install instructions for PATH exports" or something like that

Reply:
Updated.